setup/options.rst

```diff
@@ -0,0 +1,228 @@
+
+       Linear
+
+    .. figure:: _static/scaling_quality_smooth.png
+
+       Smooth Nearest Neighbour
+
+  * **Pause game when Steam overlay is open**
+
+    Requires a hardware renderer. Enables the game to be paused when using the Steam overlay.
+
+* **Uncap FPS**
+
+  When disabled, the framerate is capped at 40. When enabled, allows the framerate to run higher by generating in-between frames for smoother gameplay.
+
+* **Show FPS**
+
+  When enabled, shows a small number at the top of the screen of the current framerate.
+
+* **Use vertical sync**
+
+  When enabled, attempts to synchronize frames with the monitor's refresh rate, preventing screen tearing.
+
+* **Multithreading**
+
+  Experimental option which, when enabled, uses multiple threads to render.
+
+* **Minimise fullscreen on focus loss**
+
+  When enabled, if the mouse cursor leaves the window in fullscreen mode, minimize the game to the taskbar.
+
+
+Rendering
+---------
+
+* **Landscape Smoothing**
+
+  Enables/disables landscape smoothing on grid edges.
+
+  .. image:: _static/landscape_smoothing.gif
+
+* **Gridlines on Landscape**
+
+  Enables/disables gridlines on the landscape.
+
+  .. image:: _static/landscape_gridlines.png
+
+* **Display text on banners in uppercase**
+
+  If enabled, displays the text on banners (ride entrances, signs) in uppercase.
+
+  .. figure:: _static/banner_text_uppercase.png
+     :scale: 200 %
+
+     Uppercase text
+
+  .. figure:: _static/banner_text_lowercase.png
+     :scale: 200 %
+
+     Upper and lowercase text
+
+* **Show guest purchases as animation**
+
+  If enabled, shows when guests make a purchase as floating green text.
+
+  .. image:: _static/show_guest_purchases.png
+
+* **Transparent background for giant screenshots**
+
+  If enabled, giant screenshots will have a transparent background where the outside of the park would normally be, instead of the dark void.
+
+* **Virtual floor style**
+
+  Can be one of three values:
+
+  * *Disabled* - No virtual floor is shown when building scenery off ground level.
+  * *Clear (Transparent)* - An outline of the current building grid is shown beneath the scenery object being placed.
+  * *Glassy (Translucent)* - An outline of the current building grid is shown beneath the scenery object being placed, and the interior of the grid is a translucent olor.
+
+  .. image:: _static/virtual_floor_disabled.png
+     :scale: 70 %
+
+  .. image:: _static/virtual_floor_clear.png
+     :scale: 70 %
+
+  .. image:: _static/virtual_floor_glassy.png
+     :scale: 70 %
+
+* **Cycle day / night**
+
+  If enabled, the game will cycle between day and night cycles, causing the colour palette to change.
+
+  .. image:: _static/day_night_cycle.gif
+
+  * **Enable lighting effects**
+
+    If enabled, light-producing items such as lamps will emit a glow during nighttime and rainstorms.
+
+    * **Enable lighting effects on rides**
+
+      If enabled, some rides will emit a glow during nighttime and rainstorms
```

Comment:
```suggestion
      If enabled, some rides will emit a glow during nighttime and rainstorms.
```

There are more occurrences, but to save spam, I have not marked them all.

Reply:
All sentences should have periods at the end of them now, please double check.